pkg/adapter/awssqs/adapter_test.go

```diff
@@ -59,6 +59,11 @@ func TestPostMessage_ServeHTTP(t *testing.T) {
 				OnFailedPollWaitSecs: 1,
 			}
 
+			err := a.initClient()
```

Comment:
Slight preference for the 1-line format:

```golang
if err := a.initClient(); err != nil {
    t.Errorf("....")
}
```

Note that this changes the scope off `err` to be only within the `if` block, which should be harmless here, but does sometimes mean you can't use this format.